theme.json

```diff
@@ -0,0 +1,413 @@
+          "text": "var(--wp--custom--link--default)"
+        },
+        ":hover": {
+          "color": {
+            "text": "var(--wp--custom--link--hover)"
+          }
+        }
+      },
+      "h1": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--5--font-size)",
+          "lineHeight": "var(--wp--custom--heading--5--line-height)",
+          "fontWeight": "var(--wp--custom--heading--5--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--5--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--5--letter-spacing)"
+        }
+      },
+      "h2": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--4--font-size)",
+          "lineHeight": "var(--wp--custom--heading--4--line-height)",
+          "fontWeight": "var(--wp--custom--heading--4--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--4--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--4--letter-spacing)"
+        }
+      },
+      "h3": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--3--font-size)",
+          "lineHeight": "var(--wp--custom--heading--3--line-height)",
+          "fontWeight": "var(--wp--custom--heading--3--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--3--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--3--letter-spacing)"
+        }
+      },
+      "h4": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--2--font-size)",
+          "lineHeight": "var(--wp--custom--heading--2--line-height)",
+          "fontWeight": "var(--wp--custom--heading--2--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--2--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--2--letter-spacing)"
+        }
+      },
+      "h5": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "h6": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "button": {
+        "color": {
+          "background": "var(--wp--custom--button--primary--default)",
+          "text": "var(--wp--custom--button--primary--label)"
+        },
+        "typography": {
+          "fontSize": "clamp(1.00rem, calc(1.16rem + -0.15vw), 1.13rem)",
```

Comment:
I forgot I went rogue and added a responsive type style to buttons. We'll probably want to clean this up and have it referencing a primitive instead of declaring an arbitrary clamp function here.

Reply:
Yip, this would be part of what @smithjw1 had mentioned - there are values that are not being referenced as tokens.

Reply:
I'll just replace it with body--2--font-size since it's already using the other properties of it.